crates/bitwarden/src/vault/cipher/secure_note.rs

```diff
@@ -0,0 +1,13 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Copy, Serialize, Deserialize, Debug, JsonSchema)]
+pub enum SecureNoteType {
+    Generic = 0,
+}
+
+#[derive(Clone, Serialize, Deserialize, Debug, JsonSchema)]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+pub struct SecureNote {
```

Comment:
We're missing a view here?

Reply:
Yeah, as there wasn't anything to encrypt/decrypt here I forgot about it, added now.